main-app/app/scripts/game/services/make-move.js

```diff
@@ -0,0 +1,23 @@
+(function () {
+    'use strict';
+
+    angular.module('Tombola.Games.NoughtsAndCrosses.Game')
```

Comment:
Make Move and new game is better in one service - correctly written the methods have 90% the same code.